src/test/java/com/commercetools/sync/sdk2/services/impl/CustomObjectServiceImplTest.java

```diff
@@ -296,7 +297,7 @@ void createCustomObject_WithRequestException_ShouldNotCreateCustomObject()
     final String json = ow.writeValueAsString(errorResponse);
     when(byProjectKeyCustomObjectsPost.execute())
         .thenReturn(
-            CompletableFuture.failedFuture(
+            CompletableFutureUtils.failed(
```

Comment:
What is the benefit of using `CompletableFutureUtils.failed`? If none, it's better to use `CompletableFuture.failedFuture` from Java SDK.